tensorflow_addons/seq2seq/tests/beam_search_ops_test.py

```diff
@@ -80,26 +79,14 @@ def test_bad_parent_values_on_gpu():
         [[[0, 0, 0], [0, -1, 1], [2, 1, 2], [-1, -1, -1]]]
     )
     max_sequence_lengths = [3]
-    expected_result = _transpose_batch_time(
```

Comment:
Just curious why this test is changed?

Reply:
Cause it returns the same exception also in the `else` branch.

Reply:
Interesting, so the `options.is_custom_kernel_disabled` or `gather_tree` has a behavior change?

Reply:
Probably not, It true that it is coming from the else branch of the test but the test was not robust enough to consider the eager fallback:

https://source.cloud.google.com/results/invocations/5894af3c-7334-4e7a-9d2d-a2712f6c9cb0/log
```pyhon
        else:
>           beams = gather_tree(
                step_ids=step_ids,
                parent_ids=parent_ids,
                max_sequence_lengths=max_sequence_lengths,
                end_token=end_token,
            )

tensorflow_addons/seq2seq/tests/beam_search_ops_test.py:96:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/seq2seq/beam_search_decoder.py:239: in gather_tree
    return _beam_search_so.ops.addons_gather_tree(
<string>:68: in addons_gather_tree
    ???
<string>:123: in addons_gather_tree_eager_fallback
    ???
```

Reply:
@seanpmorgan What do you think about this fallback?

Reply:
There was still this point.